crates/node_binding/Cargo.toml

```diff
@@ -34,16 +24,23 @@ rspack_identifier = { path = "../rspack_identifier" }
 rspack_napi_shared = { path = "../rspack_napi_shared" }
 rspack_plugin_html = { path = "../rspack_plugin_html" }
 rspack_tracing = { path = "../rspack_tracing" }
-serde = { version = "1", features = ["derive"] }
-serde_json = "1"
-tokio = { version = "1.21.0", features = [
-  "rt",
-  "rt-multi-thread",
-  "macros",
-  "test-util",
-  "parking_lot",
-] }
-tracing = "0.1.34"
+
+anyhow      = { version = "1.0.65", features = ["backtrace"] }
+async-trait = "0.1.53"
+backtrace   = "0.3"
+dashmap     = "5.4.0"
+futures     = "0.3"
+once_cell   = "1"
+regex       = "1.6.0"
+serde       = { version = "1", features = ["derive"] }
+serde_json  = "1"
+tokio       = { version = "1.28.0", features = ["rt", "rt-multi-thread", "macros"] }
```

Comment:
Note: I dropped test-util and parking_lot from tokio, they are redundant for us.